### PR TITLE
Yahoo Conversational vs informational model

### PR DESCRIPTION
**Pull request by dilyararimovna**
Add `bigru_with_max_aver_pool_model` to `keras_classification_model`, add config, docs and test for yahoo conversational vs informational model (without opportunity to train, pre-trained model is available only for inference)

### CODE REVIEW

File: deeppavlov/configs/classifiers/intents_snips.json [hunk 1]
```diff
@@ -70,8 +70,8 @@
         ],
         "main": true,
         "class_name": "keras_classification_model",
-        "save_path": "{DOWNLOADS_PATH}/classifiers/intents_snips_v7/model",
-        "load_path": "{DOWNLOADS_PATH}/classifiers/intents_snips_v7/model",
+        "save_path": "{MODELS_PATH}/classifiers/intents_snips_v7/model",
+        "load_path": "{MODELS_PATH}/classifiers/intents_snips_v7/model",
```
Comment (yoptar) [inline deeppavlov/configs/classifiers/intents_snips.json]: This should be in a separate pull request

File: deeppavlov/configs/classifiers/yahoo_convers_vs_info.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
+{
+  "dataset_iterator": {
```
Comment (yoptar) [inline deeppavlov/configs/classifiers/yahoo_convers_vs_info.json]: i feel that if there is no reader, there should be no iterator and no `in_y` and `train` blocks

File: deeppavlov/configs/classifiers/intents_snips.json (hunk 1 above)
Comment (dilyararimovna) [reply deeppavlov/configs/classifiers/intents_snips.json]: ok

File: deeppavlov/configs/classifiers/yahoo_convers_vs_info.json (hunk 2 above)
Comment (dilyararimovna) [reply deeppavlov/configs/classifiers/yahoo_convers_vs_info.json]: yes. removed

File: deeppavlov/configs/classifiers/yahoo_convers_vs_info.json [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        "ref": "my_one_hotter",
+        "in": "y_pred_ids",
+        "out": "y_pred_onehot"
+      }
+    ],
+    "out": [
+      "y_pred_labels"
+    ]
+  },
+  "train": {
```
Comment (yoptar) [inline deeppavlov/configs/classifiers/yahoo_convers_vs_info.json]: Train block without dataset reader and iterator does nothing

File: deeppavlov/core/layers/keras_layers.py [hunk 4]
```diff
@@ -277,3 +278,35 @@ def compute_output_shape(self, input_shape):
...
+        seq_lengths: tensor of integers of size (batch_size, 2).
+            Each row is a pair (i, length) where length is a number of element of sequences[i] to extract
+
+    Returns:
+        tensor of shape (batch_size, dim)
+    """
+    return Lambda(lambda x: tf.gather_nd(x[0], K.cast(x[1], "int32")))([sequences, seq_lengths])
+
+
+def extracting_part_of_sequences(sequences, seq_lengths):
```
Comment (yoptar) [inline deeppavlov/core/layers/keras_layers.py]: `extracting_part_of_sequences` and `masking_sequences` are identical
Also, are you sure, you cannot write them as `Layer` inheritors?

File: deeppavlov/models/classifiers/keras_classification_model.py [hunk 5]
```diff
@@ -162,7 +160,8 @@ def _change_not_fixed_params(self, **kwargs) -> None:
                 self.opt[param] = kwargs.get(param)
         return
 
-    def pad_texts(self, sentences: List[List[np.ndarray]]) -> np.ndarray:
+    def pad_texts(self, sentences: List[List[np.ndarray]],
+                  return_lengths=False) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
```
Comment (yoptar) [inline deeppavlov/models/classifiers/keras_classification_model.py]:
```suggestion
                  return_lengths: bool = False) -> Union[np.ndarray, Tuple[np.ndarray, np.ndarray]]:
```
also, `return_lengths` is not described in docstring

File: deeppavlov/configs/classifiers/yahoo_convers_vs_info.json (hunk 3 above)
Comment (dilyararimovna) [reply deeppavlov/configs/classifiers/yahoo_convers_vs_info.json]: I will left this block with only parameter `metrics` to provide user with further understanding of their correct usage

File: deeppavlov/core/layers/keras_layers.py (hunk 4 above)
Comment (dilyararimovna) [reply deeppavlov/core/layers/keras_layers.py]: I actually do not use this functions. I will remove them

File: deeppavlov/core/layers/keras_layers.py [hunk 6]
```diff
@@ -17,6 +17,7 @@
 from keras.layers import Dense, Reshape, Concatenate, Lambda
 from keras.layers.merge import Multiply
 from keras.activations import softmax
+import tensorflow as tf
```
Comment (yoptar) [inline deeppavlov/core/layers/keras_layers.py]: `tensorflow` is not used in this file
